launch/client.py

```diff
@@ -1904,6 +1904,63 @@ def _streaming_request(
         )
         return response
 
+    def _sync_request(
```

Comment:
All of this was already in code before and got accidentally deleted somehow - adding this back now